howso/client/pandas/client.py

```diff
@@ -132,13 +128,13 @@ def get_feature_conviction(self, *args, **kwargs) -> DataFrame:
         response = super().get_feature_conviction(*args, **kwargs)
         index = []
         rows = []
-        if response.get('familiarity_conviction_addition'):
+        if  response['get']('familiarity_conviction_addition'):
```

Comment:
Not sure why `.get` as changed to `['get']`

Reply:
I was doing some regex replacements so this got caught up in it. Address by c28e6de8dded39f04ba3206735fe885f931d8d2e